wordpress-mobile/WordPress-Android
# Improving Zendesk tracked events

Partially Fixes #10540

AFAIU the Zendesk SKD for Android does not have a ready to use mechanism to notify us some of the wanted events as [iOS SDK version](https://developer.zendesk.com/embeddables/docs/ios_support_sdk/handle_events) have instead. 

Waiting a final confirmation from Zendesk support about this but meanwhile this PR aims to add the two from the events listed in the #10540 that can be added since they rely only on user interaction; namely:



- `support_new_request_viewed`: this must be triggered when a user access to the Zendesk `RequestActivity`

<p align="center">
  <img width="300" src="https://user-images.githubusercontent.com/47797566/66148622-a0e97e00-e611-11e9-8e62-cad4d5820074.png">
</p>

- `support_ticket_list_viewed`: this must be triggered when a user access to the Zendesk `RequestListActivity` (the list of user tickets)

<p align="center">
  <img width="300" src="https://user-images.githubusercontent.com/47797566/66149863-43a2fc00-e614-11e9-8073-4e6a5a1122c5.png">
</p>

## To test

**Note 1:** it can take 5 to 10 minutes or more sometimes to get the events in Tracks live view.
**Note 2:** in Tracks live view you can filter per event but also per username to narrow down the list.

- The `RequestActivity` is accessed in the `createNewTicket` function. To trigger it tap for example on `Me > Help & Support > Contact us` and check that you get the `support_new_request_viewed` event in Tracks live view.

- The `RequestListActivity` is accessed in the `showAllTickets` function. To trigger it tap on `Me > Help & Support > My tickets` and check that you get the `support_ticket_list_viewed ` event in Tracks live view.

PR submission checklist:
